src/main/java/de/tum/in/www1/artemis/service/user/UserService.java

```diff
@@ -419,7 +421,7 @@ private Optional<User> findUserInLdap(String userIdentifier, Supplier<Optional<L
 
     /**
      * Updates the user (and synchronizes its password) and its groups in the connected version control system (e.g. GitLab if available).
-     * Also updates the user groups in the used authentication provider (like {@link JiraAuthenticationProvider}.
+     * Also updates the user groups in the used authentication provider (like {@link JiraAuthenticationProvider}).
```

Comment:
Please use another example. Since we're removing the Atlassian stack, Jira is deprecated.